tests/instrumentation-framework/src/test/resources/testOpenTracingPolyglot/test.json

```diff
@@ -9,7 +9,7 @@
   "tests": [
     {
       "description" : "Context propagation",
-      "skip": false,
+      "skip": true,
```

Comment:
And I was thinking about how lucky I am that I got two build successes in a row!